doc/sphinx/source/conf.py

```diff
@@ -95,7 +95,7 @@
 
 # General information about the project.
 project = u'ESMValTool'
-copyright = u'{0}, Veronika Eyring, Axel Lauer, Mattia Righi, Martin Evaldsson et al.'.format(
+copyright = u'{0}, ESMValTool Core Development Team'.format(
```

Comment:
```suggestion
copyright = u'{0}, ESMValTool Development Team'.format(
```
The copyright lies with anyone who ever contributed some code, not just the core development team.